OpenRA.Mods.Cnc/UtilityCommands/ImportRedAlertMapCommand.cs

```diff
@@ -80,7 +80,7 @@ void UnpackTileData(MemoryStream ms)
 		static readonly string[] OverlayActors = new string[]
 		{
 			// Fences
-			"sbag", "cycl", "brik", "fenc", "wood",
+			"sbag", "cycl", "brik", "fenc", "wood", "barb",
```

Comment:
this doesn't look correct. Where did you get this from? If it's this list https://github.com/electronicarts/CnC_Remastered_Collection/blob/7d496e8a633a8bbf8a14b65f490b4d21fa32ca03/REDALERT/DEFINES.H#L1420-L1425 then barb and fenc should be swapped

Reply:
I had just appended it to the line because the index didn't seem to have an effect unlike the other array. Fixed because that indeed seems better. Thanks.